keel-sqs/src/main/kotlin/com/netflix/spinnaker/keel/sqs/SqsResourceCheckListener.kt

```diff
@@ -39,17 +43,16 @@ internal class SqsResourceCheckListener(
 
   internal fun handleMessages() {
     with(sqsClient) {
-      receiveMessage(queueUrl)
+      receiveMessages()
         .also {
-          log.debug("Got {} messages", it.messages.size)
+          if (it.isNotEmpty()) log.debug("Got {} messages: {}", it.size, it.map(Message::getMessageId))
```

Comment:
Logging is pretty noisy currently. I've also added the message ids so I can validate there's no duplicate reading going on on multiple instances in the Keel cluster.